libcxx/test/std/utilities/memory/util.smartptr/util.smartptr.shared.atomic/atomic_compare_exchange_strong.verify.cpp

```diff
@@ -0,0 +1,32 @@
+//===----------------------------------------------------------------------===//
```

Comment:
These deprecated tests are typically in `libcxx/test/libcxx/depr/depr.XXX` where XXX is the section name in the Standard. They typically have a cxxYY too in their name. (We're not very consistent in this naming but let's match that generic direction.)